circle.yml

```diff
@@ -1860,46 +1860,6 @@ jobs:
             echo "console.log('hello world')" > hello.ts
             npx tsc hello.ts --noEmit
 
-  # testing scaffolding examples and running them
-  # against example.cypress.io
```

Comment:
Do we need to record this as an action item to follow up on / add back in one way or another?

Reply:
Hm yes probably, we want to make sure all the scaffolded specs still function and work, just without all the complexity of the existing scaffolding setup.

Reply:
https://cypress-io.atlassian.net/browse/UNIFY-657

Reply:
https://cypress-io.atlassian.net/browse/UNIFY-657